infrastructures/docker/src/main/java/org/eclipse/che/workspace/infrastructure/docker/local/LocalCheInfrastructureProvisioner.java

```diff
@@ -32,20 +36,21 @@
 import javax.inject.Inject;
 import javax.inject.Named;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
+import java.util.Collections;
 import java.util.List;
+import java.util.Map;
 
+import static java.lang.String.format;
 import static org.eclipse.che.api.workspace.shared.Utils.getDevMachineName;
 
 /**
  * Infrastructure provisioner that uses volumes for workspaces projects storing because it relies on local Che usage.
  *
  * @author Alexander Garagatyi
  */
-public class LocalCheInfrastructureProvisioner extends DefaultInfrastructureProvisioner {
-    private static final List<String> SNAPSHOT_EXCLUDED_DIRECTORIES = Arrays.asList("/tmp");
+// TODO should add default RAM settings, hosts, and all other stuff from MachineStarter
+public class LocalCheInfrastructureProvisioner implements InfrastructureProvisioner {
```

Comment:
Well, it doesn't do anything heavy, so it doesn't have to be a singleton. I'll add the annotation.